python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat.py

```diff
@@ -360,8 +360,17 @@ async def main() -> None:
             pass
         elif isinstance(task, str):
             first_chat_message = TextMessage(content=task, source="user")
-        elif isinstance(task, TextMessage | MultiModalMessage | StopMessage | HandoffMessage):
+        elif isinstance(task, (TextMessage, MultiModalMessage, StopMessage, HandoffMessage)):
             first_chat_message = task
+        elif isinstance(task, list):
+            if not task:
+                raise ValueError("Task list cannot be empty")
+            if not all(isinstance(msg, (TextMessage, MultiModalMessage, StopMessage, HandoffMessage)) for msg in task):
+                raise ValueError("All messages in task list must be valid ChatMessage types")
+            first_chat_message = task[0]
```

Comment:
Putting it into the output queue won't make the messages available to the team. We need to update the `GroupChatStart` event type to use a list of messages, and inside `BaseGroupChatManager`, populate the team members with the messages, also performing the termination condition check.

Reply:
- Modified `GroupChatStart` event type in `_events.py` to support list of messages
- Updated `BaseGroupChatManager.handle_start()` to properly handle list of messages
- Added logic to populate team members with messages
- Implemented proper termination condition checking for message lists